examples/step-89/doc/results.dox

```diff
@@ -0,0 +1,92 @@
+<h1>Results</h1>
+
+<h3>Vibrating membrane: Point-to-point interpolation vs. Nitsche-type mortaring</h3>
+
+We compare the results of the simulations after the last time step, i.e. at $t=8T$.
+The $y$-component of the velocity field using Nitsche-type mortaring is depicted on the left.
+The same field using point-to-point interpolation is depicted on the right.
+
+<b>TODO: all pictures should go to the dealii website to keep the source code small</b>
+<table align="center" class="doxtable">
+  <tr>
+    <td>
+        @image html step-89-vm-cut.png "" width=60%
+    </td>
+    <td>
+        @image html step-89-vm-p2p.png "" width=60%
+    </td>
+  </tr>
+</table>
+
+Besides this, the results for the pressure and the velocity in $y$ direction
+are plotted along the horizontal line that spans from (0,0.69) to (1,0.69).
+Note, that the gap in the line plots (located at the position of the
+interface) is only related to the way Paraview displays the line plot. Using
+more sophisticated output generators resolves this. To focus on the important
+parts of the tutorial we do not want to add further lines of code only to
+remove the gap.
```

Comment:
Have you tried using the cell-locator instead of the point locator when using the plot over line in paraview?